buildscripts/kokoro/linux_presubmit.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+# This file is used for Linux builds.
+# It expects TASK environment variable is defined.
+# To run locally:
+#  ./buildscripts/kokoro/linux.sh
+
+# This script assumes `set -e`. Removing it may lead to undefined behavior.
+set -exu -o pipefail
+
+# It would be nicer to use 'readlink -f' here but osx does not support it.
+readonly OPENCENSUS_JAVA_DIR="$(cd "$(dirname "$0")"/../.. && pwd)"
+
+# cd to the root dir of opencensus-java
+cd $(dirname $0)/../..
+
+valid_tasks() {
+  echo "Valid tasks are"
+  echo ""
+  echo "- BUILD"
+  echo "- BUILD_EXAMPLES_BAZEL"
+  echo "- BUILD_EXAMPLES_GRADLE"
+  echo "- BUILD_EXAMPLES_MAVEN"
+  echo "- CHECKER_FRAMEWORK"
+  echo "- CHECK_EXAMPLES_FORMAT"
+  echo "- CHECK_EXAMPLES_LICENSE"
+  echo "- CHECK_GIT_HISTORY"
+}
+
+if [[ ! -v TASK ]]; then
+  set +x
+  echo "TASK not set in environment"
+  valid_tasks
+  exit 1
+fi
+
+case "$TASK" in
+  "CHECK_GIT_HISTORY")
+    python ./scripts/check-git-history.py
+    ;;
+  "BUILD")
+    ./gradlew clean assemble --stacktrace
+    ./gradlew check :opencensus-all:jacocoTestReport
+    ./gradlew verGJF
+    curl -s https://codecov.io/bash | bash
```

Comment:
It looks like this command didn't succeed in https://source.cloud.google.com/results/invocations/6c4b2d41-79b7-4c1f-b455-8fe121688b97/log:

```
+ curl -s https://codecov.io/bash
+ bash
  _____          _
 / ____|        | |
| |     ___   __| | ___  ___ _____   __
| |    / _ \ / _` |/ _ \/ __/ _ \ \ / /
| |___| (_) | (_| |  __/ (_| (_) \ V /
 \_____\___/ \__,_|\___|\___\___/ \_/
                              Bash-0b37652
x> No CI provider detected.
    Testing inside Docker? http://docs.codecov.io/docs/testing-with-docker
    Testing with Tox? https://docs.codecov.io/docs/python#section-testing-with-tox
    project root: .
    Yaml not found, that's ok! Learn more at http://docs.codecov.io/docs/codecov-yaml
==> Running gcov in . (disable via -X gcov)
==> Python coveragepy not found
==> Searching for coverage reports in:
    + .
    -> Found 1 reports
==> Detecting git/mercurial file structure
==> Reading reports
    + ./all/build/reports/jacoco/test/jacocoTestReport.xml bytes=1162483
==> Appending adjustments
    http://docs.codecov.io/docs/fixing-reports
    -> No adjustments found
==> Gzipping contents
==> Uploading reports
    url: https://codecov.io
    query: branch=pull_branch&commit=fa9860512b982d7d3ca02f940c050f09cca4a12e&build=&build_url=&name=&tag=&slug=census-instrumentation%2Fopencensus-java&service=&flags=&pr=&job=
    -> Pinging Codecov
HTTP 400
Please provide the repository token to upload reports via `-t :repository-token`
```

Reply:
I'll remove it for now and add it back when other commits are ready.